ada/utils.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import, division
+
+from django.conf import settings
+from temba_client.v2 import TembaClient
+
+rapidpro = None
+if settings.EXTERNAL_REGISTRATIONS_V2:
```

Comment:
I don't think we need this, maybe check if RAPIDPRO_URL and RAPIDPRO_TOKEN is available

Reply:
Fixed